data/txHandler.go

```diff
@@ -221,16 +223,27 @@ func (handler *TxHandler) asyncVerifySignature(arg interface{}) interface{} {
 }
 
 func (handler *TxHandler) processIncomingTxn(rawmsg network.IncomingMessage) network.OutgoingMessage {
+	// attempt to preallocate buffer on stack:
```

Comment:
Adding the check here will miss the SolicitedTxHandler used by the txSyncer when syncing the pools. 
Hooking this up at the backlog, both streams can be checked.

Reply:
we want to do it *before* queueing to let the backlog some breathing space